diff --git a/port/cpl_csv.h b/port/cpl_csv.h
index e3b6adc..bba54bd 100644
--- a/port/cpl_csv.h
+++ b/port/cpl_csv.h
@@ -71,7 +71,7 @@ void CPL_DLL CSVDeaccess( const char * );
 const char CPL_DLL *CSVGetField( const char *, const char *, const char *,
                                  CSVCompareCriteria, const char * );
 
-void CPL_DLL SetCSVFilenameHook( const char *(*)(const char *) );
+// void CPL_DLL SetCSVFilenameHook( const char *(*)(const char *) );
 
 CPL_C_END
 
